client-flink/common/src/main/java/org/apache/celeborn/plugin/flink/readclient/FlinkShuffleClientImpl.java

```diff
@@ -150,7 +150,6 @@ public RssBufferStream readBufferedPartition(
     } else {
       return RssBufferStream.create(
           this,
-          conf,
           flinkTransportClientFactory,
           shuffleKey,
           fileGroups.partitionGroups.get(partitionId).toArray(new PartitionLocation[0]),
```

Comment:
`fileGroups.partitionGroups.get(partitionId).toArray()` still can't keep the order by epoch. we'd better sort partitions in client. 
``` java
PartitionLocation[] partitionLocations = fileGroups.partitionGroups.get(partitionId).toArray(new PartitionLocation[0]);
Arrays.sort(partitionLocations, Comparator.comparingInt(PartitionLocation::getEpoch));
```